requests/src/test/scala/weco/api/requests/services/SierraRequestsServiceTest.scala

```diff
@@ -30,150 +26,6 @@ class SierraServiceTest
     with SierraServiceFixture {
 
   describe("SierraService") {
-    describe("getAccessConditions") {
```

Comment:
These tests are replaced by the extra case here: https://github.com/wellcomecollection/catalogue-api/pull/242/files#diff-9ed61b263ed01f0a064581dfb2ce9ab7ecdaa54dc15a08b555c08fbf1b6e6cb7R239